charts/exporter/values.yaml

```diff
@@ -0,0 +1,41 @@
+annotations: {}
+replicaCount: 1
+imagePullPolicy: Always
```

Comment:
I don't think `Always` is a good default. What about `IfNotPresent`?

Reply:
Thx. I've probably forgot to update it, after debugging.
Fixed it in faef101f25abe418baacb2c17a3a803e735a526d